checkov/terraform/graph_builder/local_graph.py

```diff
@@ -365,7 +365,7 @@ def _get_dest_module_path(self, curr_module_dir: str, dest_module_source: str, d
                 if 'Non-relative patterns are unsupported' in str(e):
                     return ""
                 raise e
-        return os.path.realpath(dest_module_path)
+        return os.path.abspath(dest_module_path)
```

Comment:
@YaaraVerner do we want to wrap it with the env var?

Reply:
@achiar99 no, it's a problem also for the current implementation